source/math/FloatComplexVectorList.ooc

```diff
@@ -161,4 +174,32 @@ FloatComplexVectorList: class extends VectorList<FloatComplex> {
 			result add(value)
 		result
 	}
+	_fastFourierTransformBufferSize: static func (inputSize: Int) -> Int {
+		logValue := inputSize as Float log() / 2.0f log()
```

Comment:
Yes, put them in a separate pull request. If you want, I can merge that one first so you can edit this one to use them. I don't have a list of functions I'd like to add, no. but I've seen functions lying around in various classes that only deal with `Float`s and that should probably be moved there.